package.json

```diff
@@ -38,12 +45,26 @@
     ]
   },
   "devDependencies": {
+    "@babel/preset-flow": "^7.14.5",
+    "@babel/preset-react": "^7.14.5",
+    "@babel/runtime": "^7.15.4",
     "babel-eslint": "^10.1.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.6",
```

Comment:
Why are we using Enzyme for testing? A better option would be to use the "React testing library".